src/button/button.scss

```diff
@@ -11,6 +11,9 @@ $iui-button-padding-large: $iui-xs * 6;
 
 @mixin iui-button {
   --_iui-button-active-stripe-inset: initial;
+  --_iui-button-label-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-2));
```

Comment:
I don't think this is just `label-color`, it is more like `foreground-color` because it also applies to the icon fill

Reply:
Changed to `--_iui-button-foreground-color`.